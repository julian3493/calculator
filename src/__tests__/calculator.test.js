import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../pages/calculator';

it('it test the render Home page', () => {
  const calculator = renderer
    .create(<Calculator />)
    .toJSON();
  expect(calculator).toMatchSnapshot();
});
