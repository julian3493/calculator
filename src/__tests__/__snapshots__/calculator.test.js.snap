// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it test the render Home page 1`] = `
<div
  className="container"
>
  <h2>
    Lets do some math
  </h2>
  <div
    className="App"
  >
    <div
      className="display"
    >
      <h1 />
    </div>
    <div
      className="panel"
    >
      <div
        className="row"
      >
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            AC
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            +/-
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            %
          </button>
        </div>
        <div
          className="btn btnColor"
        >
          <button
            className="btn btnColor"
            name="button"
            onClick={[Function]}
            type="button"
          >
            รท
          </button>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            7
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            8
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            9
          </button>
        </div>
        <div
          className="btn btnColor"
        >
          <button
            className="btn btnColor"
            name="button"
            onClick={[Function]}
            type="button"
          >
            X
          </button>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            4
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            5
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            6
          </button>
        </div>
        <div
          className="btn btnColor"
        >
          <button
            className="btn btnColor"
            name="button"
            onClick={[Function]}
            type="button"
          >
            -
          </button>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            1
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            2
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            3
          </button>
        </div>
        <div
          className="btn btnColor"
        >
          <button
            className="btn btnColor"
            name="button"
            onClick={[Function]}
            type="button"
          >
            +
          </button>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="btn btn0"
        >
          <button
            className="btn btn0"
            name="button"
            onClick={[Function]}
            type="button"
          >
            0
          </button>
        </div>
        <div
          className="btn"
        >
          <button
            className="btn"
            name="button"
            onClick={[Function]}
            type="button"
          >
            .
          </button>
        </div>
        <div
          className="btn btnColor"
        >
          <button
            className="btn btnColor"
            name="button"
            onClick={[Function]}
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
